examples/crna-kitchen-sink/storybook/stories/Welcome/index.js

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable import/no-extraneous-dependencies, import/no-unresolved, import/extensions */
+
+import React, { PropTypes } from 'react';
+import { View, Text } from 'react-native';
+
+export default class Welcome extends React.Component {
+  styles = {
+    wrapper: {
+      flex: 1,
+      padding: 24,
+      justifyContent: 'center',
+    },
+    header: {
+      fontSize: 18,
+      marginBottom: 18,
+    },
+    content: {
+      fontSize: 12,
+      marginBottom: 10,
+      lineHeight: 18,
+    },
+  };
+
+  showApp(event) {
+    event.preventDefault();
```

Comment:
You do not typically need to `.preventDefault()` in React Native. That said I don't see the `showApp` function used anywhere in this component... so couldn't this method just be removed?